README.md

```diff
@@ -100,137 +45,118 @@ For more information on the importance of a professional README for your reposit
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
+- Transactions: Users can create transactions, which are associated with a category. The category can be anything, such as "Food," "Entertainment," or "Transportation."
 
-## 🚀 Live Demo <a name="live-demo"></a>
+- Categories: Users can create and manage categories. This allows them to track their expenses more granularly.
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://google.com)
```

Comment:
- [ ] [REQUIRED] I've tried to reach out to you on Slack to help you to fix it but know response Please note that For this capstone it is verry important to deploy your app as it is part of requirements and also as developer you need to know Application Deployment is one of the most important stages of the software development process as the strategy that is used to build, test, and deploy will directly impact how fast an application can respond to changes in constituent preferences or requirements, and most importantly.  👍